src/components/Download/Download.jsx

```diff
@@ -0,0 +1,150 @@
+    const response = await fetch(url);
+    const data = await response.json();
+    let downloadUrl;
+
+    if (Platform == "Win32") {
+      downloadUrl = data.assets && data.assets[3].browser_download_url;
+      window.location.href = downloadUrl;
+    } else if (Platform == "Linux x86_64") {
+      downloadUrl = data.assets && data.assets[0].browser_download_url;
+      window.location.href = downloadUrl;
+    } else if (Platform == "MacIntel") {
+      downloadUrl = data.assets && data.assets[1].browser_download_url;
+      window.location.href = downloadUrl;
+    }
+  };
+
+  return (
+    <div>
+      <h2 className="text-center my-4">Download Terasology Launcher</h2>
+      <div className="download-underline"></div>
+      <div className="m-5">
+        <Row className="justify-content-center">
+          <Col lg="2" md="3" sm="4" xs="4" className="pr-0">
+            <h5 className="my-3 ">Select OS: </h5>
+          </Col>
+          <Col lg="7" md="6" sm="8" xs="8" className="download-dropdown">
+            <Input
+              type="select"
+              name="select"
+              id="exampleSelect"
+              style={{ height: "40px", fontSize: "15px" }}
+              onChange={(e) => {
+                const selectedos = e.target.value;
+                setPlatform(selectedos);
+              }}
+            >
+              <option
+                value="Win32"
+                selected={
+                  window.navigator.platform == "Win32" ? "selected" : ""
+                }
+              >
+                Windows (64-bit)
+              </option>
+              <option
+                value="Linux x86_64"
+                selected={
+                  window.navigator.platform == "Linux x86_64" ? "selected" : ""
+                }
+              >
+                Linux (64-bit)
+              </option>
+              <option
+                value="MacIntel"
+                selected={
+                  window.navigator.platform == "MacIntel" ? "selected" : ""
+                }
+              >
+                macOS
+              </option>
+            </Input>
+          </Col>
+          <Col lg="3" md="9">
+            <div className="text-center font-weight-bold btn-primary download-btn">
+              <NavLink className="text-white" onClick={() => download()}>
+                <IconContext.Provider
+                  value={{ size: "1em", className: "download" }}
+                >
+                  <FaDownload />
+                </IconContext.Provider>
+                Download
+              </NavLink>
+            </div>
+          </Col>
+        </Row>
+      </div>
+
+      <div className="download-underline"></div>
+      <div className="my-5 ">
+        <h2 className=" m-3 download-system-requirement">
+          System Requirements
+        </h2>
+        <div className="my-5">
+          <Table className="download-table my-4" borderless>
+            <thead>
```

Comment:
We have our grid system ( react-bootsrap ) , so is it still right to use <table> stuff ?
It would more uniform is we can use one layout system throughout ?

Reply:
Actually,ModuleSite is using  reactstarp for more information  you can refer this to "https://reactstrap.github.io/components/tables/#app"

Reply:
you are right @ryuk156 .
And in reactsstrap too we have https://reactstrap.github.io/components/layout/ this layout system.
I was just going in favor os using this layout rather than using table so as to maintain uniform layout sytem.

btw table layout is cool too :+1: